src/apps/watchfaces/OswAppWatchfaceFitnessAnalog.cpp

```diff
@@ -0,0 +1,323 @@
+    hal->gfx()->setTextLeftAligned();
+    hal->gfx()->setTextCursor(CENTER_X - 70, 170);
+    OswAppWatchfaceDigital::dateOutput(yearInt, monthInt, dayInt);
+
+    hal->gfx()->setTextSize(4);
+    hal->gfx()->setTextLeftAligned();
+    hal->gfx()->setTextCursor(CENTER_X - 35, CENTER_Y);
+
+    hal->gfx()->printDecimal(hour, 2);
+    hal->gfx()->print(":");
+    hal->gfx()->printDecimal(minute, 2);
+
+    hal->gfx()->setTextSize(2);
+    hal->gfx()->setTextLeftAligned();
+    hal->gfx()->setTextCursor(215, CENTER_Y);
+    hal->gfx()->printDecimal(second,2);
+
+    if (!OswConfigAllKeys::timeFormat.get()) {
+        const char am[] = "AM";
+        const char pm[] = "PM";
+        hal->gfx()->setTextCursor(215, 130);
+        if (afterNoon) {
+            hal->gfx()->print(pm);
+        } else {
+            hal->gfx()->print(am);
+        }
+    }
+
+#if OSW_PLATFORM_ENVIRONMENT_TEMPERATURE == 1
+/*    
+    printStatus("Temperature", String(hal->environment()->getTemperature() + String("C")).c_str());
+    for(auto& d : *OswTemperatureProvider::getAllTemperatureDevices())
+        printStatus((String("  ") + d->getName()).c_str(), String(d->getTemperature() + String("C")).c_str());
+*/
+    hal->gfx()->setTextSize(2);
+    hal->gfx()->setTextLeftAligned();
+    hal->gfx()->setTextCursor(DISP_W * 0.2f, DISP_H * 0.2f);
+    hal->gfx()->print(hal->environment()->getTemperature(), 1);
+    hal->gfx()->print("C");
+#endif
+}
+
+const char* OswAppWatchfaceFitnessAnalog::getAppId() {
+    return OswAppWatchfaceFitnessAnalog::APP_ID;
+}
+
+const char* OswAppWatchfaceFitnessAnalog::getAppName() {
+    return "AFIT"; //LANG_FITNESS;
+}
+
+void OswAppWatchfaceFitnessAnalog::onStart() {
+    OswAppV2::onStart();
+    OswAppWatchface::addButtonDefaults(this->knownButtonStates);
+
+    // Report that we support short presses on all buttons
+    this->knownButtonStates[Button::BUTTON_SELECT] = ButtonStateNames::SHORT_PRESS ;
+    this->knownButtonStates[Button::BUTTON_UP] = ButtonStateNames::SHORT_PRESS;
+    this->knownButtonStates[Button::BUTTON_DOWN] = ButtonStateNames::SHORT_PRESS;
+
+    // Here is a snippet to also "support" double presses (on BUTTON_SELECT) - note that this WILL DELAY the reporting of any short press events on that button (as it may needs to wait for the second press)
+    this->knownButtonStates[Button::BUTTON_SELECT] = (OswAppV2::ButtonStateNames) (this->knownButtonStates[Button::BUTTON_SELECT] | OswAppV2::ButtonStateNames::DOUBLE_PRESS); // OR to set the bit
+    this->knownButtonStates[Button::BUTTON_UP] = (OswAppV2::ButtonStateNames) (this->knownButtonStates[Button::BUTTON_UP] | OswAppV2::ButtonStateNames::DOUBLE_PRESS); // OR to set the bit
+    this->knownButtonStates[Button::BUTTON_DOWN] = (OswAppV2::ButtonStateNames) (this->knownButtonStates[Button::BUTTON_DOWN] | OswAppV2::ButtonStateNames::DOUBLE_PRESS); // OR to set the bit
+
+    this->lastTime = time(nullptr); // use
+    printf("xxx onStart\n");
```

Comment:
please avoid using `printf` directly: use `OSW_LOG_XXX` or (if you must) `Serial.println` instead